queries.sql

```diff
@@ -59,3 +59,37 @@ FROM animals GROUP BY species;
 SELECT species, AVG(escape_attempts) FROM animals   
 WHERE  date_of_birth >= '1990/01/01' AND date_of_birth <= '2000/12/31'
 GROUP BY species;
+
+-- queries for the owners table
+
+SELECT full_name, name FROM owners 
+JOIN animals ON owners.id = animals.owner_id
+WHERE full_name = 'Melody Pond';
+
+SELECT a.name AS animal_name, s.name AS species FROM animals AS a
+JOIN species s on s.id = a.species_id 
+WHERE s.name = 'Pokemon';
+
+SELECT owners.full_name, animalS.name
+FROM owners
+LEFT JOIN animals ON owners.id = animals.owner_id;
+
+SELECT species.name, count(species.name) as amount FROM animals 
+JOIN species ON animals.species_id = species.id
+GROUP BY species.name;
+
+SELECT o.full_name AS owner, a.name AS animal_name, s.name AS species
+FROM animals AS a
+JOIN species AS s on a.species_id = s.id
+JOIN owners AS o ON a.owner_id = o.id
+WHERE s.name = 'Digimon' AND  o.full_name = 'Jennifer Orwell';
+
+SELECT a.name AS animal_name, o.full_name AS owner, a.escape_attempts 
+FROM animals AS a
+JOIN owners AS o ON a.owner_id = o.id
+WHERE o.full_name = 'Dean Winchester' AND a.escape_attempts = 0;
+
+SELECT full_name, COUNT(full_name) AS amount FROM owners
+JOIN animals ON animals.owner_id = owners.id
+GROUP BY full_name
+ORDER BY amount DESC LIMIT 1;
```

Comment:
- [ Required ] I think it would be better if you also add screenshots to your pull request for these four queries as requested by the previous reviewer. The query screenshots help to determine whether you have actually run the queries with success. Kindly add them to the description of the readme file.

Reply:
Hi @Felix45 
I added screenshots